scripts/migrate.js

```diff
@@ -94,6 +95,22 @@ const run = async () => {
     rimraf.sync(NAV_DIR);
     rimraf.sync(DICTIONARY_DIR);
 
+    logger.info('Fetching redirects');
+
+    const orderedRedirects = (docs) => {
+      return docs.reduce((accum, curr) => {
+        let { to, from } = curr.redirect;
+        to = to.replace(/\//g, '');
```

Comment:
[nit] It looks like you are looking up the [node by its ID]https://github.com/newrelic/docs-website/pull/500/files#diff-9c154e1779b4a100a8a1713152855b8554a563f6360c2393bacbb55f465747f2R38). Rather than needing to look it up via `node<id>`, could you just replace the `node` text here to make it easier to lookup?

```suggestion
        to = to.replace(/\//g, '').replace('node', '');
```

This makes it easier to lookup later:

```js
redirect[doc.docId]
```

Reply:
i wasn't sure if i should leave `node` in the key name, because the redirect data also contains taxonomy term redirects, too. so if/when we wanted to get those out, i thought it might be useful to know a key is a Drupal node or a taxonomy term.